app/services/statements/statements_query.rb

```diff
@@ -0,0 +1,28 @@
+module Statements
+  class StatementsQuery
+    def initialize(lead_provider:, params: {})
+      @lead_provider = lead_provider
+      @cohorts = String(params[:cohort]).split(",")
+      @updated_since = params[:updated_since]
+    end
+
+    def statements
+      statements = Statement.includes(:cohort).where(lead_provider:)
+      statements = statements.where(cohorts: { start_year: cohorts }) if cohorts.any?
+      statements = statements.where("updated_at >= ?", @updated_since) if @updated_since.present?
```

Comment:
`updated_since` is a `attr_reader` and can we use range instead:
```ruby
      statements = statements.where(updated_at:, updated_since..) if updated_since.present?
```

Also, we do some date parsing check with `updated_since` on ecf: https://github.com/DFE-Digital/early-careers-framework/blob/main/app/services/api/concerns/filter_updated_since.rb, lets do that here.

Reply:
I was wondering if we could keep filtering to the filtering ticket? as we can do the whole slice then?